code/API_definitions/CAMARA/NUMBER_VERIFICATION_API.md

```diff
@@ -88,9 +88,10 @@ Following table provides an overview of common error names, codes, and messages
 |2	|401 |	UNAUTHENTICATED |	"Request not authenticated due to missing, invalid, or expired credentials" |
 |3	|403 |	PERMISSION_DENIED |	"Client does not have sufficient permissions to perform this action" |
 |4	|403 |	NUMBER_VERIFICATION.USER_NOT_AUTHENTICATED_BY_IP |	"Client must authenticate via IP to use this service" |
```

Comment:
If we say that we use *network authentication* for Number Verify should we give 403 NUMBER_VERIFICATION.USER_NOT_AUTHENTICATED_BY_IP?